.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,21 +2,24 @@
 | ------------- | ---
 | Branch?       | 7.4 for features / 6.4, 7.2, or 7.3 for bug fixes <!-- see below -->
```

Comment:
```suggestion
| Branch?       | 7.4 for features / 6.4, 7.2, or 7.3 for bug fixes
```

Could we also remove this? I don't see which section it actually points at

Reply:
OK. Removed. Thanks.

Reply:
see below => "Contributor guidelines" where how the choice should be made is explained
I'm fine removing if you think it's not needed. I don't have stats to decide one way or another :)